cmd/broker/filter/main.go

```diff
@@ -80,29 +70,44 @@ func main() {
 		log.Fatal("Error building kubeconfig", err)
 	}
 
-	kubeClient := kubernetes.NewForConfigOrDie(cfg)
+	ctx, _ = injection.Default.SetupInformers(ctx, cfg)
+	kubeClient := kubeclient.Get(ctx)
+
+	config, err := sharedmain.GetLoggingConfig(ctx)
```

Comment:
there is already a `cfg`, and now this `config`. Might be clearer if we rename it to `loggingCfg` or `loggingConfig` ?